src/main/scala/akka/persistence/eventstore/query/scaladsl/EventStoreReadJournal.scala

```diff
@@ -63,6 +75,25 @@ class EventStoreReadJournal(system: ExtendedActorSystem, config: Config)
       if (to > Int.MaxValue) EventNumber.Last else eventNumber(to))
   }
 
+  private def eventsByTag(tag: String, from: Long, infinite: Boolean): Source[EventEnvelope, akka.NotUsed] = {
+    val streamId = EventStream.System(s"ce-$tag")
```

Comment:
I do not think that this is `obvious`, for somebody it should be `tag = streamId`, for others `tag = prefix + streamId`, this case implemented for those using projection creating `$ce-tag` stream.